### DISCUSSION TITLE
老大救救，我把网址复制进去以后就显示这个，浏览器也没有登陆的弹窗出来。等了很久也重装了一次还是这样，我看其他人都能进到下一步，是我漏了什么吗

### DISCUSSION THREAD

**Discussion by 4675746SRDT** [mikumifa/cppTickerBuy#18] [提问专区]
<img width="699" alt="屏幕截图 2024-09-26 175052" src="https://github.com/user-attachments/assets/e34d6c85-8e0d-4598-8095-f62ddfd3db0f">

**Comment by mikumifa** [comment]
命令行里面输入账号密码

**Reply by 4675746SRDT** [reply]
谢谢老大！!可以了！我一个人瞎研究半天
